experiments/extract_all_data.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# $1 must match the path to the DreamCoder PLDI artifact, and must
+# not include a trailing /
+ARTIFACT_PATH=$1
```

Comment:
add a check/abort if no argument is given

Reply:
Added in [ae809ff](https://github.com/mlb2251/stitch/pull/87/commits/ae809ffa5311b86a57d85d2aaba1bab554683a2d)